roles/openstack_helm_octavia/defaults/main.yml

```diff
@@ -56,7 +56,7 @@ openstack_helm_octavia_heartbeat_key: "{{ undef(hint='You must specify a Octavia
 # .. envvar:: openstack_helm_octavia_management_subnet_cidr [[[
 #
 # Octavia management subnet (CIDR)
-openstack_helm_octavia_management_subnet_cidr: "172.24.0.0/22"
+openstack_helm_octavia_management_subnet_cidr: "{{ octavia_management_subnet_cidr | default('172.24.0.0/22') }}"
```

Comment:
You should be able to simply add "openstack_helm_octavia_management_subnet_cidr" into your variables and you should be good to go.

The reason we've used the other model of variable was because it was a variable used across multiple roles.